src/GameClient.ts

```diff
@@ -34,7 +34,7 @@ export class GameClient extends Client {
         .then(endpoints => {
             return super.open({
                 authToken: options.authToken,
-                url: endpoints[0].address,
+                url: options.url || endpoints[0].address,
```

Comment:
Well, actually. Running the standalone service it exposes the a "fake" discovery endpoint which discovers itself. So alternately the option could be passed as an API base URL or something, which could be reused later if, for example, shortcode OAuth support is integrated into the library. (The standalone tetrisd service also exposes fake shortcode endpoints)